shortnames.conf

```
@@ -56,6 +57,7 @@
   "rhel7" = "registry.access.redhat.com/rhel7"
   "rhel7.9" = "registry.access.redhat.com/rhel7.9"
   "rhel-atomic" = "registry.access.redhat.com/rhel-atomic"
+  "rhel-bootc" = "registry.redhat.io/rhel9/rhel-bootc"
```

Let's not tie the `rhel` alias to `rhel9`
```suggestion
  "rhel9-bootc" = "registry.redhat.io/rhel9/rhel-bootc"
```

---

I would like to have rhel-bootc be equivalent to rhel-bootc:latest and that to shift when RHEL10 is released.  But I guess this is a decent compromize for now.